monitor/widgets/src/main/java/org/datacleaner/monitor/scheduling/widgets/SchedulePanel.java

```diff
@@ -70,19 +77,25 @@
     Button executeButton;
 
     @UiField
+    InlineLabel latestExecutionLabel;
+    
+    @UiField
     FlowPanel alertsPanel;
+    
 
     public SchedulePanel(final ClientConfig clientConfig, final ScheduleDefinition schedule,
             final SchedulingServiceAsync service) {
         super();
 
         _clientConfig = clientConfig;
         _schedule = schedule;
+        _service = service; 
```

Comment:
When I reformat the code in my IDE the trailing space from this line is removed.